### PR TITLE
Fix for mail.google.com (gmail)

### PR DESCRIPTION
**Pull request by rakleed**
- Invert category icons.
- Darken the background of the pop-up side menu.
- Remove obsolete rule.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -7509,6 +7508,12 @@ CSS
 .buj {
     background-image: url(//ssl.gstatic.com/ui/v1/icons/mail/rfr/density_compact_v1_1x.png) !important;
 }
+div[class*="bym"][role="navigation"] {
+    background-color: rgb(25 27 28) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:7512]: Could we something that isn't hardcoded?

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -7509,6 +7508,12 @@ CSS
...
+}
+.byl .nZ.aBP {
+    background-color: rgba(255,255,255,0.05) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:7515]: For this one as well.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (rakleed) [reply src/config/dynamic-theme-fixes.config]: Fixed.

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (rakleed) [reply src/config/dynamic-theme-fixes.config:7516]: The closest to the current version I found the variable `--lt-color-gray-700: #2F3237;`, but still it turns out not a gray shade, but a brown one, and it looks worse than the current version. Therefore, it is better to leave it as it is.

**Review by Gusted — APPROVED** [review]

**Pull request merged**